swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
+
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+        
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        try:
+            self.inventory.remove(item)
+            return item
+        except Exception:
+            return False
```

Comment:
Great use of error handling to deal with `remove` not finding the item. We should try to make the error we handle as specific as possible in order to avoid silencing unexpected errors. In this case, we can test that the actual error raised by `remove` is `ValueError`, which is what we should use in our `except` clause.

```python
    def remove(self, item):
        try:
            self.inventory.remove(item)
            return item
        except ValueError:
            return False
```